tests/fast-integration/skr-test/provision/deprovision-skr.js

```diff
@@ -0,0 +1,19 @@
+const {deprovisionSKR} = require('../../kyma-environment-broker');
+const {keb, kcp} = require('../provision/provision-skr');
+
+
+async function deprovisionSKRInstance(options, timeout) {
+  try {
+    await deprovisionSKR(keb, kcp, options.instanceID, timeout);
+  } catch (e) {
+    throw new Error(`before hook failed: ${e.toString()}`);
```

Comment:
Wouldn't this be rather called in an `after` block ?

Reply:
I changed it to `De-provisioning failed:`